zql/docs/processors/README.md

```diff
@@ -154,6 +155,58 @@ ssl   1521912240.189735 CSbGJs3jOeB6glWLJj 10.47.7.154 27137     52.85.83.215 44
+| **Description**           | Return results under a single record definition that unifies the field and type information across all records in the query result. |
+| **Syntax**                | `fuse`                                            |
+| **Required<br>arguments** | None                                              |
+| **Optional<br>arguments** | None                                              |
+| **Limitations**           | Because `fuse` must make a first pass through the data to assemble the unified schema, results from queries that use `fuse` will not begin streaming back immediately.<br><br>If the query result is too large, an error message `"fuse processor exceeded memory limit"` will be returned. Issue [zq/1320](https://github.com/brimsec/zq/issues/1320) tracks the removal of this limitation. |
+| **Developer Docs**        | https://godoc.org/github.com/brimsec/zq/proc/fuse |
+
+#### Example:
+
+Let's say you'd started with table-formatted output of both `stats` and `weird` events:
+
+```zq-command
+zq -f table 'ts < 1521911721' stats.log.gz weird.log.gz
+```
+
+#### Output:
+```zq-output
+_PATH TS                PEER MEM PKTS_PROC BYTES_RECV PKTS_DROPPED PKTS_LINK PKT_LAG EVENTS_PROC EVENTS_QUEUED ACTIVE_TCP_CONNS ACTIVE_UDP_CONNS ACTIVE_ICMP_CONNS TCP_CONNS UDP_CONNS ICMP_CONNS TIMERS ACTIVE_TIMERS FILES ACTIVE_FILES DNS_REQUESTS ACTIVE_DNS_REQUESTS REASSEM_TCP_SIZE REASSEM_FILE_SIZE REASSEM_FRAG_SIZE REASSEM_UNKNOWN_SIZE
+stats 1521911720.600725 zeek 74  26        29375      -            -         -       404         11            1                0                0                 1         0         0          36     32            0     0            0            0                   1528             0                 0                 0
+_PATH TS                UID                ID.ORIG_H   ID.ORIG_P ID.RESP_H      ID.RESP_P NAME                             ADDL NOTICE PEER
+weird 1521911720.600843 C1zOivgBT6dBmknqk  10.47.1.152 49562     23.217.103.245 80        TCP_ack_underflow_or_misorder    -    F      zeek
+weird 1521911720.608108 -                  -           -         -              -         truncated_header                 -    F      zeek
+weird 1521911720.610033 C45Ff03lESjMQQQej1 10.47.5.155 40712     91.189.91.23   80        above_hole_data_without_any_acks -    F      zeek
+weird 1521911720.742818 Cs7J9j2xFQcazrg7Nc 10.47.8.100 5900      10.129.53.65   58485     connection_originator_SYN_ack    -    F      zeek
+```
+
+Here a `stats` event was the first record type to be printed in the results stream, so the preceding header row describes the names of its fields. Then a `weird` event came next in the results stream, so a header row describing its fields was printed. This presentation accurately conveys the heterogeneous nature of the data, but the frequent interruption of data rows with different headers is distracting and may make the output difficult to process in downstream tools.
```

Comment:
I'm not sure "distracting" is the right motivation.  Why not say that certain tooling like SQL inserts or csv output cannot handle heterogeneous record types so fuse allows you to combine them into a unified schema.

Reply:
I was on the fence about covering much CSV here because I know the output is due to change when we address #1326, so I was sticking with tables, which I admit is pretty banal. But your feedback is certainly valid so I've committed a change that shows CSV and mentions SQL, and the automation will remind whoever addresses #1326 to fix the example output.

(A closing note on this change in case someone circles back to this PR at a later date. The `zq` command line that causes the `csv output requires uniform records but different types encountered` message is intentionally _not_ run through the output checker. This is because the automation that checks these examples appears to fail any test that returns a non-zero exit code. Ideally we'd enhance the automation to allow the marking of a `zq` command where we _expect_ this to happen, but we want to check that the output and error message that make it through before the exit don't change. However I didn't wanna hold this up by enlisting a Go developer to write that enhancement, so I'm pushing forward with what we've got.)